STEERING-COMMITTEE.md

```diff
@@ -162,13 +120,157 @@ There are two ways to raise issues to the steering committee for decision:
+for nominations and exceptions will be closed.  The election will be open for
+voting not less than two weeks and not more than four.  The results of the
+election shall be announced within one week of closing the election.  New
+Steering Committee members will take office on December 1 of each year.
+
+### Election Officer(s)
+
+Six weeks or more before the election, the Steering Committee will appoint
+between one and three Election Officer(s) to administer the election.  Elections
+Officers will be Knative community members in good standing who are eligible to
+vote, are not running for Steering in that election, and can make a public
+promise of impartiality.  They will be responsible for:
+
+* Making all announcements associated with the election
+* Preparing and distributing electronic ballots
+* Judging exception requests
+* Assisting candidates in preparing and sharing statements
+* Tallying voting results according to the rules in this charter
+
+### Eligibility to Vote
+
+Anyone who has at least 50 contributions in the last 12 months is eligible to
+vote in the Steering election. Contributions are defined as opening PRs,
+reviewing and commenting on PRs, opening and commenting on issues, writing
+design docs, commenting on design docs, helping people on slack, participating
+in working groups, and other efforts that help advance Knative the project.
+
+[This
+dashboard]((https://knative.teststats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year)
+shows only GitHub based contributions and does not capture all the contributions
+we value. We expect this metric not to capture everyone who should be eligible
+to vote. If a community member has had significant contributions over the past
+year but is not captured in the [stats.knative.dev
+dashboard](https://knative.teststats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year),
+they will be able to submit an exception form to the Elections Officer(s) who
+will then review and determine whether this member should be marked as an
+exception.  All exceptions, and the reasons for them, will be recorded in a log
+that will be available to Steering and the TOC.
+
+All eligible voters will be captured at
+knative/community/steering-elections/$YEAR/voters.md and the voters’ guide will
+be captured at knative/community/steering-elections/$YEAR/README.md, similar to
+the kubernetes election process and identical to the TOC election process.
+
+We are committed to an inclusive process and will adapt future eligibility
+requirements based on community feedback.
+
+### Candidate Eligibility
+
+Community members must be eligible to vote in order to stand for election (this
+includes voters who qualify for an exception). Candidates may self-nominate or
+be nominated by another eligible member. The approximate time commitment of a
+Steering member is around 5 hours per week.
```

Comment:
Is this true for steering? It looks copy-pasted from TOC, so want to verify.

Reply:
We approximated 5 hours, since steering specific tasks will be distributed across 7 members.

Reply:
Assuming that TOC's and steering's tasks are not overlapping, the time commitment might be also an argument to not merge TOC and steering (as discussed above) since this would increase the time commitment to ~ 11 hours in average per member (assuming 7 seats for a combined board, (8 * 5 + 5 * 7) / 7 = 10.7)

Reply:
I removed the sentence in question :)